### ISSUE TITLE
JavaFX WebView no longer works with Google Maps JS sdk

### ISSUE THREAD

**Issue by Erel2** [rterp/GMapsFX#207]
It looks like something has changed recently with the Maps JS SDK and it doesn't work properly with JavaFX WebView.
The map is empty and every tile shows the message "Sorry, we have no imagery here.". Tested with various versions of JavaFX from v8 to v21.
Tested on Windows.

It happens with this implementation as well as online sites based on the maps sdk.

**Comment by Cdavison-development** [comment]
seeing the same, quite unfortunate.

**Comment by timoreichen** [comment]
Hi.  I have reported the bug, see link. I wrote Kevin Rushford (no answer) but for whatever reason, the door to the internet, the webview is not getting any attention for improvements. Its just not working working correctly for many pages. Maybe one here with more influence can change that somehow. My 50 cents, its cause of interconnection to jxbrowser behind the scenes. Dont see any other explanation that webviev is that bad.

https://bugs.openjdk.org/browse/JDK-8331640?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel

**Comment by nehabluedot** [comment]
Map not loading for my project as well, waiting for an update from Java team:
<img width="698" alt="Screenshot 2024-09-09 at 3 36 04 pm" src="https://github.com/user-attachments/assets/2ccfda82-a2c0-4892-9299-ebecca6ad1f3">

**Comment by Landgraf13** [comment]
I found a solution - [source](https://www.b4x.com/android/forum/threads/google-map-error.161434/#:~:text=My%20intuition%20was%20right%2C%20version%203.56%20should%20have%20been%20specified%20and%20not%203.exp%20which%20uses%20jGoogleMaps)

_"My intuition was right, version **3.56** should have been specified and not **3.exp** which uses jGoogleMaps"_
I corrected the maps.html for myself and it finally worked

**Comment by mattiasbe** [comment]
>  version **3.56** should have been specified

Yes v3.56 works fine for me too as a short term workaround, but will not be available forever .

This picture about version availablity is from https://developers.google.com/maps/documentation/javascript/versions

<img width="395" alt="image" src="https://github.com/user-attachments/assets/6f706db4-3657-4e42-b310-ce305d410c8e">

**Comment by garten-nack** [comment]
Hello, has anyone found a solution? I have exactly the same issue. When I try to use version 3.57, the map does not load.
My guess is that the new 3.57 version loads WebP tiles, and JavaFX is not able to display them.

**Comment by danielwilbanks** [comment]
> Hello, has anyone found a solution? I have exactly the same issue. When I try to use version 3.57, the map does not load. My guess is that the new 3.57 version loads WebP tiles, and JavaFX is not able to display them.

No solution that I know of, my guess is that this is sort of dead now. I was able to switch my app over to leaflet instead, it doesn't work in a webview with the latest version but if you downgrade it to 1.8.0 then it does work. Unsure how long I can get away with that but for now it does what I need it to.
